velox/functions/prestosql/tests/ArgGeneratorTest.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <gtest/gtest.h>
+#include "velox/expression/fuzzer/tests/ArgGeneratorTestUtils.h"
+#include "velox/functions/prestosql/fuzzer/DivideArgGenerator.h"
+#include "velox/functions/prestosql/fuzzer/FloorAndRoundArgGenerator.h"
+#include "velox/functions/prestosql/fuzzer/MultiplyArgGenerator.h"
+#include "velox/functions/prestosql/fuzzer/PlusMinusArgGenerator.h"
+
+using namespace facebook::velox;
+namespace {
+
+class ArgGeneratorTest : public testing::Test {};
+
+TEST_F(ArgGeneratorTest, plusMinus) {
+  auto signature =
```

Comment:
Instead of hard-coding function signature here, can we fetch it from the registry?

Reply:
Added this function to fetch signature from the registry.
>   std::vector<const exec::FunctionSignature*> getDecimalSignatures(
      const std::string& functionName) const {